_docs-sources/foundations/iac-foundations/labels-tags.md

```diff
@@ -0,0 +1,264 @@
+
+Gruntwork recommends using shorthand where possible to avoid exceeding limits for resource names as they may very across AWS services. For example, the `gruntwork` organization name can be shortened to `gw`, the `production` environment can be shortened to `prd`, and the `us-west-2` AWS region can be shortened to `uw2`.
+
+
+| Descriptor | Example | Rationale | Required |
+| ---------- | ------- | -------- | -------- |
+| organization | `acme` | Assists with making globally unique resource names (e.g., S3 buckets) | Yes |
+| environment | `prd` | The environment (dev, stage, prod, etc) in which the service is running | Yes |
+| application | `coolapp` | Identify which application the resource belongs to | Yes |
+| identifier | `compute` | Identify specific workloads within an application | Yes |
+| region | `uw2` | The AWS region in which the service is running | Yes |
+
+As an example, if Acme Co was running an application named `coolapp` on AWS EC2 in a production environment in us-west-2, we would label all EC2 instances `acme-prd-coolapp-compute-uw2`. Similarly if we were using AWS RDS as the database layer for the same application, we would label the associated RDS instance `acme-prd-coolapp-db-uw2`.
+
+## Tags Overview
+
+Tags are key value pairs that act as metadata for organizing your AWS resources. Tags are often used by operators and developers to identify workload owners (e.g., a specific team), the application that a resource belongs to, the environment in which an application is running, and more. Finance teams often make use of teams to map cost and usage to specific Cost Centers or Business Units. For more information on tags, see the [official documentation](https://docs.aws.amazon.com/tag-editor/latest/userguide/tagging.html).
+
+### Tag key recommendations
+
+Gruntwork recommends prefixing all custom tag keys with the name of your organization. For example - the company acme co would start all of their tag keys with `acmeco`. This will allow you to distinguish tags specific to your organization from tags that may be automatically applied to your by AWS.
+
+### Recommended tags
+
+The table below contains recommend tag keys and the description for the value you should give to the key. These keys are beneficial for both locating and filtering for resources as an operator or developer as well as being used as [cost allocation tags](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/cost-alloc-tags.html) to assist finance teams with associating costs to specific business units or applications.
+
+
+| Key Name   | Example Key | Value Description |
+| ---------- | ----------- | ----------- |
+| Department | `acmeco:Department` | The organizational department to which the owning team belongs. Useful for rolling up costs per department in the organization |
```

Comment:
I'm used to seeing colons used as the delimiter for tags, e.g. `key:value`. Here, the colon is just a _part_ of the key, correct. Is that potentially confusing? Is that a standard convention?